docker-compose.yml

```diff
@@ -59,6 +62,8 @@ x-app_settings: &app_settings
   depends_on:
     - server
   command: ./scripts/install_and_run.sh
+  environment:
```

Comment:
This can be removed from the docker-compose file since this PR doesn't include the frontend part

Reply:
done